Adding -no-canonical-prefixes to cflags breaks the build in lunar by
preventing the compiler from finding basic system headers, e.g.

  In file included from ../../base/third_party/dynamic_annotations/dynamic_annotations.c:35:
  /usr/include/stdlib.h:32:10: fatal error: 'stddef.h' file not found
  #include <stddef.h>
           ^~~~~~~~~~
  1 error generated.

--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1492,7 +1492,7 @@ config("compiler_deterministic") {
   # paths to the tools it invokes. We don't want this because we don't
   # really need it and it can mess up the goma cache entries.
   if (is_clang && (!is_nacl || is_nacl_saigo)) {
-    cflags += [ "-no-canonical-prefixes" ]
+    #xtradeb#cflags += [ "-no-canonical-prefixes" ]
 
     # Same for links: Let the compiler driver invoke the linker
     # with a relative path and pass relative paths to built-in
